From f5ee4fda8e81211456e953d1f4e5cce212eac505 Mon Sep 17 00:00:00 2001
From: ZhongWen <b47623@freescale.com>
Date: Sun, 29 Sep 2013 11:34:50 +0800
Subject: [PATCH 31/52]  ENGR00278171 wfd-sink: update wfd-sink apk

 Do some optimization for the apk:
    Solve the problem that White screen show first when switch to black GUI if open Sink App

Signed-off-by: ZhongWen <b47623@freescale.com>
---
 WfdSink/AndroidManifest.xml      | 1 -
 WfdSink/res/layout/sink_main.xml | 1 -
 WfdSink/res/values/styles.xml    | 2 +-
 3 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/WfdSink/AndroidManifest.xml b/WfdSink/AndroidManifest.xml
index 20465f4..a5a5c1e 100755
--- a/WfdSink/AndroidManifest.xml
+++ b/WfdSink/AndroidManifest.xml
@@ -3,7 +3,6 @@
     package="com.freescale.wfdsink"
     coreApp="true"
     android:sharedUserId="android.uid.system" >
-
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.ACCESS_ALL_EXTERNAL_STORAGE" />
diff --git a/WfdSink/res/layout/sink_main.xml b/WfdSink/res/layout/sink_main.xml
index b2fbd3f..c033291 100755
--- a/WfdSink/res/layout/sink_main.xml
+++ b/WfdSink/res/layout/sink_main.xml
@@ -18,7 +18,6 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:layout_gravity="center"
-    android:background="#000000"
     android:orientation="vertical" >
 
     <RelativeLayout
diff --git a/WfdSink/res/values/styles.xml b/WfdSink/res/values/styles.xml
index f3ea816..93c6707 100755
--- a/WfdSink/res/values/styles.xml
+++ b/WfdSink/res/values/styles.xml
@@ -13,7 +13,7 @@
     </style>
 
     <!-- Application theme. -->
-    <style name="AppTheme" parent="AppBaseTheme">
+    <style name="AppTheme" parent="@android:style/Theme.Black">
         <!-- All customizations that are NOT specific to a particular API-level can go here. -->
     </style>
 	
-- 
1.8.0

